components/elm/cime_config/config_pes.xml

```diff
@@ -523,4 +523,42 @@
       </pes>
     </mach>
   </grid>
+  <grid name="a%ne30np4.pg2_l%r05_oi%null_r%r05_g%mpas.gis20km_w%null_z%null_m%EC30to60E2r2">
```

Comment:
I'm always confused by what should go into which config files.  Does it make sense to put this in the ELM pes config file?

Reply:
I don't know. This was where Jon suggested it should go. Note that the default layouts for BG configs is in `./cime_config/allactive/config_pesall.xml`.